crepecake/build.gradle

```diff
@@ -14,11 +14,16 @@ repositories {
 }
 
 sourceSets {
+    File sharedTestDir = file('src/shared-test/java')
+    test {
+        java.srcDir sharedTestDir
+    }
     integrationTest {
         java {
             compileClasspath += main.output + test.output
```

Comment:
doesn't this already give integ-test access to anything from test?

Reply:
Oh yea lemme just put the shared stuff in test then.